### PR TITLE
Add whitehall specific email and atom links to taxonomy views

### PR DESCRIPTION
**Pull request by bevanloon**
Collections is going to be responsible for rendering routes at `/world/{country}`. These will take the place of the current `/government/world/{country}` routes that are served by Whitehall. 

However, we will still be relying on Whitehall to provide email subscriptions for the `world_locations`. 

This PR adds some custom logic to create links to the correct path for pages that live at `/world` so that we can continue to lean on the Whitehall email subscription functionality.

[Trello](https://trello.com/c/2Xk5rE3k/197-add-whitehall-email-and-atom-links-to-taxonomy-views)

Current email links look like this:
screen shot 2017-06-14 at 16 11 11

Emails links for `/world` will look like this:
screen shot 2017-06-14 at 16 09 07

### CODE REVIEW

File: test/unit/email_helper_test.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
+  test "should return a valid whitehall .atom url in the form /government/{url}.atom" do
+    define_singleton_method(:request) do
+      return OpenStruct.new(fullpath: "/world/blefuscu")
+    end
+
+    expected_atom_url = Plek.new.website_root + "/government/world/blefuscu.atom"
+
+    puts Plek.new.website_root
```
Comment (davidbasalla) [inline test/unit/email_helper_test.rb]: This can probably be removed?

Comment (bevanloon) [inline test/unit/email_helper_test.rb]: grr - thought I got all those

**Comment by davidbasalla** [comment]
Can we add a screenshot in the PR description of how this looks on the page? 👀

**Comment by davidbasalla** [comment]
Cool, looks good! 👍

**Comment by bevanloon** [comment]
Screenshots added

**Pull request merged**